packages/edit-site/src/components/actions/index.js

```diff
@@ -33,7 +33,9 @@ export const trashPostAction = {
 		return status !== 'trash';
 	},
 	hideModalHeader: true,
-	RenderModal: ( { item: post, closeModal } ) => {
+	RenderModal: ( { items: posts, closeModal } ) => {
+		// Todo - handle multiple posts
```

Comment:
Is this something we plan to address here?

Reply:
This PR is already big so I planned to implement bulk actions for pages in a follow-up.